README.md

```diff
@@ -1 +1,63 @@
-# spark-kubernetes-operator
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
```

Comment:
We don't need to have this in `README.md`.

Please refer
- https://github.com/apache/spark/blob/master/README.md?plain=1

Reply:
removed per https://github.com/apache/spark-kubernetes-operator/pull/2/commits/f0a2e742dc98cec12bcde166223a778128b038cd